### PR TITLE
Outputs reworked

### PR DESCRIPTION
**Pull request by LegenJCdary**

### CODE REVIEW

File: source/modules/misc/arguments.py [hunk 1]
```diff
@@ -85,6 +89,10 @@ def validate_arguments(arguments: Namespace) -> dict:
                   f" Available commands are: {sub_string}.{PRINT_END}")
             sys.exit(57)
```
Comment (LegenJCdary) [inline source/modules/misc/arguments.py:102]: Exit only from main and instead return None/integer?

File: setup.cfg [hunk 2]
```diff
@@ -34,6 +34,7 @@ packages =
     ansible_deployer
     ansible_deployer.modules
     ansible_deployer.modules.configs
+    ansible_deployer.modules.misc
```
Comment (cinek810) [inline setup.cfg:37]: Am I correct that this commit shouldn't introduce any functionali change? That's something that is usually good to emphasize in commit message.

File: source/modules/misc/arguments.py (hunk 1 above)
Comment (cinek810) [reply source/modules/misc/arguments.py:115]: I don't think we should change that approach before v1

File: source/modules/globalvars.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 PRINT_END = '\033[0m'
 OPTION_EXPANSION = {
     "infra": "infrastructure",
-    "raw_output": "raw_runner_output",
+    "raw_output": "runner_raw_output",
```
Comment (cinek810) [inline source/modules/globalvars.py:10]: there is no need to introduce a breaking change in user interface, just keep it as it was

**Review by cinek810 — CHANGES_REQUESTED** [review]
We can add new option that looks more standard, but we should keep the old one in place too. I'm not sure it goes the optimal way, I thought we should udplicate the file descriptor and just connect  it to users stdout, but... maybe it's something to be considered in next version. For v1... let's give it a try.

**Review by cinek810 — APPROVED** [review]

**Pull request merged**